src/ngraph_builder.cc

```diff
@@ -1717,34 +1717,36 @@ static Status TranslateFusedBatchNormOp(
   std::shared_ptr<ng::Node> ng_batch_norm;
 
   if (tf_is_training) {
-    ng_batch_norm = make_shared<ng::op::BatchNormTraining>(tf_epsilon, ng_scale,
-                                                           ng_offset, ng_input);
+    ng_batch_norm = make_shared<ng::op::BatchNormTraining>(
+        ng_input, ng_scale, ng_offset, tf_epsilon);
 
-    shared_ptr<ngraph::Node> ng_y, ng_mean, ng_variance;
-    ng_y = make_shared<ng::op::GetOutputElement>(ng_batch_norm, 0);
-    ng_mean = make_shared<ng::op::GetOutputElement>(ng_batch_norm, 1);
-    ng_variance = make_shared<ng::op::GetOutputElement>(ng_batch_norm, 2);
+    shared_ptr<ngraph::Node> ng_y_out, ng_mean_out, ng_variance_out;
```

Comment:
I must misunderstand about the training op orders. In ngraph, shouldn't the output order be `{gamma, beta, input}`? Could you please explain this a little bit?

Reply:
So this PR was to sync with this PR in ngraph, which reorders batch norm: https://github.com/NervanaSystems/ngraph/pull/2046#issuecomment-440072078

But apparently that has been closed, and will come later, so I suppose we don't have to do anything now.